html-forms/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form</h1>
+    <form>
+      <div>Email<br /></div>
+      <input type="text" name="email" size="20"/>
+      <div>Why is coding cool?</div>
```

Comment:
This should be the `label` element, not a `div`. This is the text your user would click to try and put focus on the input it pairs with.